src/RazorSdk/Targets/Sdk.Razor.CurrentVersion.targets

```diff
@@ -44,6 +44,21 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!-- Resolve the TFM-specific attributes conditionally. -->
   <Choose>
+    <When Condition=" '$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND $([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)', '10.0')) ">
+      <PropertyGroup>
+        <_TargetingNETCoreApp30OrLater>true</_TargetingNETCoreApp30OrLater>
+        <_TargetingNET50OrLater>true</_TargetingNET50OrLater>
+        <_TargetingNET60OrLater>true</_TargetingNET60OrLater>
+        <_TargetingNET70OrLater>true</_TargetingNET70OrLater>
+        <_TargetingNET80OrLater>true</_TargetingNET80OrLater>
+        <_TargetingNET90OrLater>true</_TargetingNET90OrLater>
+        <_TargetingNET100OrLater>true</_TargetingNET100OrLater>
+        <UseRazorSourceGenerator Condition="'$(Language)' == 'C#' AND '$(UseRazorSourceGenerator)' == '' ">true</UseRazorSourceGenerator>
+        <RazorLangVersion Condition="'$(RazorLangVersion)' == '' ">9.0</RazorLangVersion>
```

Comment:
I assume this is intentionally 9.0 not 10.0 because we want to switch to newer lang version only later in the cycle?

Reply:
Correct; I also expect that putting 10.0 here would break until the razor compiler knows what that is.